README.rst

```diff
@@ -74,6 +76,18 @@ For migration purposes, it is possible to transfer all public playlists of a use
 
     spotify_to_ytmusic all <spotifyuserid>
 
+Transfer liked tracks of the Spotify user
+-----------------------------------------
+
+**You must you oAuth authentification for transfering liked songs.**
+
+.. code-block::
+
+   spotify_to_ytmusic liked
+
+This command will open browser where you should give access to your account (if you havn't done that before). 
```

Comment:
This is not quite correct? The user needs to run `setup` before and choose `use_oauth` correctly, unless I missed something

Reply:
User needs to run `setup` for sure but authentication process only starts when we run command that start transferring. Maybe we should describe this behaviour in other section of readme?